Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Components/Objects/QuerySpeckleObjects.cs

```diff
@@ -136,10 +135,19 @@ protected override void SolveInstance(IGH_DataAccess dataAccess)
     // Set output objects
     for (int i = 0; i < Params.Output.Count; i++)
     {
+      // determine output values based on parameter type
       List<SpeckleWrapper> outputValues =
-        i == 0 ? filteredObjects : _filterDict[Filters[i - 1]].Select(o => (SpeckleWrapper)o).ToList();
-      List<IGH_Goo> outputGoos = outputValues.Select(o => o.CreateGoo()).ToList();
-      if (targetCollectionWrapper?.Topology is string topology && !string.IsNullOrEmpty(topology))
+        i == 0
```

Comment:
can you read this ternary operator mess?

I recommend redoing it to just have if statements.